gcc/riscv64.patch

```diff
@@ -26,24 +26,31 @@ index 1952b31..37569f0 100644
    python
    zstd
  )
-@@ -79,7 +75,6 @@ build() {
+@@ -79,7 +75,6 @@
        --mandir=/usr/share/man
        --infodir=/usr/share/info
        --with-bugurl=https://bugs.archlinux.org/
 -      --with-build-config=bootstrap-lto
        --with-linker-hash-style=gnu
        --with-system-zlib
        --enable-__cxa_atexit
-@@ -94,7 +89,7 @@ build() {
+@@ -94,7 +89,6 @@
        --enable-link-serialization=1
        --enable-linker-build-id
        --enable-lto
 -      --enable-multilib
-+      --disable-multilib
```

Comment:
unnecessary rearrange

Reply:
I just want to align enables and disables. It doesn't matter.